speck.js

```diff
@@ -5,75 +5,58 @@
   Thatsweet produce sjs syntax file, that can be compiled into
   testSpec.js files via sweet.js ans a set of ad-hoc macros.
 */
+
 var fs = require('fs');
 var path = require('path');
-// var acorn = require('acorn');
 var comments = require('./parsing/parse-comments.js');
+var extract = require('./parsing/comment-conversion.js');
+var tapeTemps = require('./templates/tape/tape-templates.js');
+var tempUtils = require('./templates/template-utils.js');
 
-// Get the list of the files to be parsed
-var files = process.argv.slice(2);
-
+// Parse command-line arguments
+var args = process.argv.slice(2);
+var testFW = args[0];
+var testPath = args[1];
+var files = args.slice(2);
 
 // Create i/o streams for each file
 files.forEach(function(fileName) {
 
+  // Look in /src for files to parse (may want to refactor later for more flexibility)
   var rStream = fs.createReadStream(path.join(__dirname, fileName));
-  // TODO: Writing output path need to be fixed
-  // var writeStream = fs.createWriteStream('./srcSpecs/testSpec-' + fileName);
-  var data = '';
 
+  // Gather data from file being read in
+  var data = '';
   rStream.on('data', function(chunk) {
     data += chunk;
   });
 
-  // Once each readable stream buffer is complete,
-  // parse it and write the result in the writable stream
+  // Readable stream buffer is complete- data now ready for use
   rStream.on('end', function() {
-    var parsedBuffer = comments.parse(data);
-    console.log(parsedBuffer.tests);
-
-    // console.dir(parsedStream.comments[1].range);
-
-
-
-
-    // writeStream.write(parsedStream.comments[0].value);
-
-    // var cmds = parsedStream.comments[0].value
-    //   .substring(6)
-    //   .split('\'').join('').split(',')
-    //   .map(function(str) {
-    //     return str.trim();
-    //   });
 
-    // // String sanitization [COWBOY STYLE]
-    // // spec, assertions, assertionType, assertionTitle, test, expected
-    // var spec = cmds[0];
-    // var assertions = cmds[1];
-    // var assertionType = cmds[2];
-    // var assertionTitle = cmds[3];
-    // var test = cmds[4] + cmds[5];
-    // var expected = cmds[6];
+    // Get SpeckJS comments from file data (Nick)
+    var tests = comments.parse(data).tests;
+    var readyTestsToWrite = [];
```

Comment:
Very small thing but I'd perhaps rename this to `testsReadyToWrite` so it's more readable.